gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/JobController.groovy

```diff
@@ -29,14 +29,6 @@ class JobController {
   @Autowired
   JobService jobService
 
-  @ApiOperation(value = "Get jobs", response = List.class)
-  @RequestMapping(value = "/applications/{applicationName}/jobs", method = RequestMethod.GET)
-  List getJobs(@PathVariable String applicationName,
```

Comment:
i can't find this used anywhere... and even if it were, clouddriver doesn't have this EP... maybe i am misreading something, please correct me if I am wrong

Reply:
can we also do a quick check from the logs for the usage ?

Reply:
yup, didn't see anything in logs